javasource/oql/implementation/OQL.java

```diff
@@ -108,7 +108,7 @@ public static Long countRowsOQL(IContext context, String statement, Long amount,
 					if (!found) {
 						throw new NullPointerException("Could not find result association " + columnSchema.getName() + " in target object.");
 					}
-				} else {
```

Comment:
That's interesting -- in https://github.com/ArjenLammers/oqlmodule/pull/7#issuecomment-555019090 @ArjenLammers says that null is a valid value!

Reply:
I think @JelleDekker has a valid point. If a value is null, but should be mapped to an association, it is in fact treated as an attribute. Then it will never find the attribute in the object.
However, the proposed fix is affecting this in a wrong manner, because null values will then always be skipped.

Reply:
I guess it would be better to have another check to see if it's an association or not? Or can we simply set it explicitly to null/empty regardless whether it's an association or attribute when the value is null?

Reply:
Hi this behavior was improved in 2.3 and should be handled now as described (checking if the null value is an association).